src/main/mondrian/rolap/aggmatcher/AggStar.java

```diff
@@ -1121,6 +1128,19 @@ public void print(final PrintWriter pw, final String prefix) {
                 pw.print("): ");
                 pw.print(generateRollupString(sqlQuery));
             }
+
+            @Override
+            public String generateExprString(SqlQuery query) {
+                String exprString = super.generateExprString(query);
+                RolapAggregator rollupAggregator = getRollupAggregator();
+                if (rollupAggregator instanceof BaseAggor) {
+                    BaseAggor agg = (BaseAggor) rollupAggregator;
```

Comment:
@lucboudreau, I'm trying to solve a specific case of agg-tables. BaseAggor is declared to be the class for aggregators over agg-tables (https://github.com/pentaho/mondrian/blob/master/src/main/mondrian/rolap/RolapAggregator.java#L265). I cannot figure out how to cram desired changes into ```RolapAggregator``` and even see no sense in it. Do you have any ideas how to implement the solution w/o similar casts?